docs/usage.md

```diff
@@ -13,6 +13,30 @@ General process:
 4. set the environment variables ([Containerfile example](#write-the-dockerfile-or-containerfile))
 5. run the build ([container build example](#build-the-container))
 
+## Configuration
+
+There are several parameters with default values set. See the [config.py](cachi2/core/config.py) module for defaults.
+Configuration can be changed by using a config file:
+
+```shell
+ cachi2 --config-file config.yaml fetch-deps --source ./my-repo gomod
+```
+
+Location of config file is in the root of the repository or use relative path to root. In the file you can set only specific parameters that you need to change.
```

Comment:
> Location of config file is in the root of the repository

What do you mean by this? There is no default config file

> or use relative path to root

You can use any path you want, relative or absolute. And it would be relative to the current directory, not to to the repo root (the --source directory)

Reply:
I wanted to write where the file should be placed, but when I look at it now, I don't think it's necessary.